demisto_sdk/commands/common/constants.py

```diff
@@ -449,7 +450,8 @@
 
 YML_SCRIPT_REGEXES: List[str] = [
     PACKS_SCRIPT_YML_REGEX,
-    PACKS_SCRIPT_NON_SPLIT_YML_REGEX
+    PACKS_SCRIPT_NON_SPLIT_YML_REGEX,
+    PACKS_SCRIPT_TEST_PLAYBOOK
```

Comment:
where do we use this?
do we want to validate those in the validate command?

Reply:
This is used in the schema validation.
Why wouldn't we want to validate the script used for test playbooks?

Reply:
just making sure we are not running BC checks on those files